stake-o-matic/src/validator_list.rs

```diff
@@ -1,23 +1,30 @@
 solana_sdk::pubkeys!(
     testnet_validators,
     [
+        "121CY7X4snhJcTEJAWG1Q2psae3uSGzXHL",
```

Comment:
This pubkey doesn't look right. It only base58 decodes to 25 bytes

Reply:
Oh hey cool, macro expansion in CI agrees :)

Reply:
Thanks @CriesofCarrots !